lib/teleterm/clusters/cluster_auth.go

```diff
@@ -277,3 +281,127 @@ func (c *Cluster) processAuthResponse(ctx context.Context, key *client.Key, resp
 
 	return nil
 }
+
+// PwdlessLogin processes passwordless logins for this cluster.
+func (c *Cluster) PwdlessLogin(ctx context.Context, stream api.TerminalService_LoginPasswordlessServer) error {
```

Comment:
Excluding docs and so on, it seems that we generally tend to use the full `passwordless` form more so I'd be for using it here too.

```
$ rg passwordless -g '!docs' -g '!*.pb.go' -g '!*.js' -g '!*_pb.d.ts' -g '!*.md' | wc -l
     445
$ rg pwdless -g '!docs' -g '!*.pb.go' -g '!*.js' -g '!*_pb.d.ts' -g '!*.md' | wc -l
      72
```

Reply:
+1 for full spelling in public APIs, well spotted.